README.md

```diff
@@ -7,36 +7,38 @@ We target the latest stable version of TypeScript, note that because we want to
+| [Cypress](#cypress-tsconfigjson)                                     | [`@tsconfig/cypress`](https://npmjs.com/package/@tsconfig/cypress)                   |
+| [Deno](#deno-tsconfigjson)                                           | [`@tsconfig/deno`](https://npmjs.com/package/@tsconfig/deno)                         |
+| [Docusaurus v2](#docusaurus-v2-tsconfigjson)                         | [`@tsconfig/docusaurus`](https://npmjs.com/package/@tsconfig/docusaurus)             |
+| [Ember](#ember-tsconfigjson)                                         | [`@tsconfig/ember`](https://npmjs.com/package/@tsconfig/ember)                       |
+| [Next.js](#nextjs-tsconfigjson)                                      | [`@tsconfig/next`](https://npmjs.com/package/@tsconfig/next)                         |
+| [Node LTS](#node-lts-22-tsconfigjson)                                | [`@tsconfig/node-lts`](https://npmjs.com/package/@tsconfig/node-lts)                 |
+| [Node 10](#node-10-tsconfigjson)                                     | [`@tsconfig/node10`](https://npmjs.com/package/@tsconfig/node10)                     |
+| [Node 12](#node-12-tsconfigjson)                                     | [`@tsconfig/node12`](https://npmjs.com/package/@tsconfig/node12)                     |
+| [Node 14](#node-14-tsconfigjson)                                     | [`@tsconfig/node14`](https://npmjs.com/package/@tsconfig/node14)                     |
+| [Node 16](#node-16-tsconfigjson)                                     | [`@tsconfig/node16`](https://npmjs.com/package/@tsconfig/node16)                     |
+| [Node 17](#node-17-tsconfigjson)                                     | [`@tsconfig/node17`](https://npmjs.com/package/@tsconfig/node17)                     |
+| [Node 18](#node-18-tsconfigjson)                                     | [`@tsconfig/node18`](https://npmjs.com/package/@tsconfig/node18)                     |
+| [Node 19](#node-19-tsconfigjson)                                     | [`@tsconfig/node19`](https://npmjs.com/package/@tsconfig/node19)                     |
+| [Node 20](#node-20-tsconfigjson)                                     | [`@tsconfig/node20`](https://npmjs.com/package/@tsconfig/node20)                     |
+| [Node 21](#node-21-tsconfigjson)                                     | [`@tsconfig/node21`](https://npmjs.com/package/@tsconfig/node21)                     |
+| [Node 22](#node-22-tsconfigjson)                                     | [`@tsconfig/node22`](https://npmjs.com/package/@tsconfig/node22)                     |
+| [Node 23](#node-23-tsconfigjson)                                     | [`@tsconfig/node23`](https://npmjs.com/package/@tsconfig/node23)                     |
+| [Node with TypeScript](node-with-typescript-ts-58-only-tsconfigjson) | [`@tsconfig/node-ts`](https://npmjs.com/package/@tsconfig/node-ts)                   |
```

Comment:
```suggestion
| [Node with TypeScript](#node-with-typescript-ts-58-only-tsconfigjson) | [`@tsconfig/node-ts`](https://npmjs.com/package/@tsconfig/node-ts)                   |
```

Looks like there was a small typo here causing the link [to fail and 404](https://github.com/tsconfig/bases/blob/main/node-with-typescript-ts-58-only-tsconfigjson). Stumbled upon this as I was just curious what this new config was. I'll have a PR out shortly to fix this

Reply:
> I'll have a PR out shortly to fix this

Fixed in #305